File: HealthcareSurvey/Models/SurveyCategory.cs
namespace HealthcareSurvey.Models
{
    public enum SurveyCategory
    {
        StaffCourtesy,
        WaitTimes,
        FacilityCleanliness,
        
        // We might add more categories, e.g., BillingExperience, TelehealthQuality, etc.
        // Also, we want to allow dynamic weighting in the future.
    }
}

File: js/src/models/surveyCategory.js
/**
 * Enum for survey categories
 * @readonly
 * @enum {number}
 */
const SurveyCategory = {
    StaffCourtesy: 0,
    WaitTimes: 1,
    FacilityCleanliness: 2,
    
    // We might add more categories, e.g., BillingExperience, TelehealthQuality, etc.
    // Also, we want to allow dynamic weighting in the future.
};

module.exports = SurveyCategory;